app/controllers/glimr_cases_controller.rb

```diff
@@ -0,0 +1,17 @@
+class GlimrCasesController < ApplicationController
```

Comment:
I know GLiMR is here to stay, but should we call this something else? Because if ever we were to create cases in another way, we would then just have to make changes in `CaseCreator`.